Rakefile

```
@@ -5,6 +5,8 @@ require 'rspec/core/rake_task'
 require 'rake_compiler_dock'
 require 'yaml'
 
+require_relative 'ext/re2/recipes.rb'
```

Is the `.rb` significant here or could we simplify?

```suggestion
require_relative 'ext/re2/recipes'
```